packages/babel-core/src/api/browser.js

```diff
@@ -1,112 +0,0 @@
-/* eslint max-len: 0 */
```

Comment:
Also could you elaborate on moving `api/node.js` to `index.js`? I tried a refactor/rename in associated files and unit tests but it lead to breaking build.